test_scripts/API/ButtonSubscription/041_Subscribe_CUSTOM_BUTTON_in_case_HmiCap_file_not_supported_Success.lua

```diff
@@ -0,0 +1,39 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0205-Avoid_custom_button_subscription_when_HMI_does_not_support.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check that SDL send SubscribeButton RPC with 'CUSTOM_BUTTON' parameter during registering App in case <button> is supported by HMI
+-- but not supported by hmi_capabilities.json
+------------------------------------------------------------------------------------------------------------------------
+-- Preconditions:
+-- 1) CUSTOM_BUTTON is missing in the hmi_capabilities.json
+-- 2) SDL and HMI are started
+-- 3) HMI supported CUSTOM_BUTTON (SDL receives Buttons.GetCapabilities response from HMI with supported CUSTOM_BUTTON)
+-- In case:
+-- 1) Mobile app starts registration
+-- SDL does:
+-- 1) Send Buttons.SubscribeButtons ( CUSTOM_BUTTON ) to HMI
+-- 2) Wait request Buttons.SubscribeButtons( SUCCESS ) from HMI
```

Comment:
```suggestion
-- 2) Wait request Buttons.SubscribeButtons( SUCCESS ) from HMI
```
-- 2) Wait response from HMI
-- 3) Receive Buttons.SubscribeButton(SUCCESS)
-- 4) Not send OnHashChange with updated hashId to mobile app